man/fi_cntr.3.md

```diff
@@ -121,7 +121,16 @@ struct fi_cntr_attr {
   and/or receives -- which are counted may be restricted using control
   flags when binding the counter and the endpoint.  Counters increment
   on all successful completions, separately from whether the operation
-  generates an entry in an event queue.
+  generates an entry in an event queue.  This includes truncated messages.
```

Comment:
I have concerns that the "this includes truncated messages" for FI_CNTR_EVENTS_COMP breaks the existing API. Specifically, looking at the fi_domain manpage, there is this section.

```
Rx Buffer Overrun
This refers to buffers posted to receive incoming tagged or untagged messages, with the behavior defined from the viewpoint of the sender. The behavior for handling received messages that are larger than the buffers provided by the application is provider specific. Providers may either truncate the message and report a successful completion, or fail the operation. For datagram endpoints, failed sends will result in the message being dropped. For reliable endpoints, send operations may complete successfully, yet be truncated at the receive side. This can occur when the target side buffers received data until an application buffer is made available. The completion status may also be dependent upon the completion model selected byt the application (e.g. FI_DELIVERY_COMPLETE versus FI_TRANSMIT_COMPLETE).
```

While a truncation at the sender may be considered a success, truncation at the receive is typically reported via `struct fi_cq_err_entry`. Does this mean truncation at receive is an error event? @j-xiong thoughts?

Reply:
Agree, this clause should be removed.